VERSION

```diff
@@ -1 +1 @@
-4.0.0
+4.1.0
```

Comment:
Version is mentioned in comments in the examples:
https://github.com/msoni-tamr/terraform-aws-emr/blob/master/examples/static-hbase/main.tf#L47

May be worth searching `4.0.0` for any other uses.